NEWS

```diff
@@ -1,3 +1,11 @@
+25-Nov-2020 Ignition 2.8.0
+
+  Features:
+
+    - Support .d directory for base configs
```

Comment:
This doesn't affect users, only packagers, so even though it's a feature I'd be inclined to put it in the `Changes` section so we don't overemphasize it.

Also, we're not only supporting the new thing, we're dropping the old thing.  Maybe `Switch system base config from single file to to .d directory`?